komodo/lint_upgrade_proposals.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+
+from komodo.yaml_file_types import KomodoException, RepositoryFile, UpgradeProposalsFile
+
+
+def verify_package_versions_exist(
+    upgrade_proposals: UpgradeProposalsFile, repository: RepositoryFile
+) -> None:
+    releases_with_upgrades_counter = 0
+    for _release_version, proposed_package_upgrades in upgrade_proposals.yml.items():
+        if proposed_package_upgrades is None:
+            continue
+        releases_with_upgrades_counter += 1
+        errors = []
+        for (
+            upgrade_proposals_package,
+            upgrade_proposals_package_version,
+        ) in proposed_package_upgrades.items():
+            try:
+                repository.validate_package_entry(
+                    upgrade_proposals_package, upgrade_proposals_package_version
+                )
+                print(
+                    f"Found package: '{upgrade_proposals_package}' with version"
+                    f" {upgrade_proposals_package_version} in repository"
+                )
+            except KomodoException as e:
+                errors.append(e.error)
+        if errors:
+            raise SystemExit(" \n ".join(errors))
```

Comment:
whitespace before `\n` should not be needed. Is the next `space` giving an intended visual effect indenting the output?

Reply:
I do not remember why I added it, probably for some visual reasons. Removing it now.